keyboards/wilba_tech/zeal65/keymaps/zyber/keymap.c

```diff
@@ -0,0 +1,38 @@
+// ZyBeRs layout for Zeal65
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```